### ISSUE TITLE
Join forces with babel-minify?

### ISSUE THREAD

**Issue by ahutchings** [escompress/escompress#10]
I found another project on github that also minifies using a collection of babel transforms. https://github.com/boopathi/babel-minify

It looks like you each have your own mangle transform - there may be some other work that can be avoided by joining forces.

**Comment by hzoo** [comment]
We released https://github.com/babel/babili and @boopathi has been helping a lot already

**Comment by imranolas** [comment]
@hzoo: That's great to see. With so much momentum behind babili it's clearly worth supporting in preference to keeping escompress alive.

@sheepsteak: Is it time to discontinue escompress?

**Comment by hzoo** [comment]
Yeah, I actually should have reached out sooner (there was nothing secret about the repo, just wanted to release when it was more stable). But it's still pretty early days and we need the help 😄

**Comment by sheepsteak** [comment]
Sounds great! @ahutchings, sorry for not responding sooner. I haven't checked this repo for a while as I've been busy on other things.

@hzoo I'd love to help on babili and I've been looking forward to seeing what you guys were doing since I heard you were doing this. To be honest, it's one of the reasons I stopped working on this really.

I've noticed you have a Webpack plugin but we could contribute Rollup and Browserify plugins if that's any help? We did do a mangler but the one in babili seems a lot more robust haha.

**Comment by hzoo** [comment]
We can always use more test cases and testing. Yeah adding those plugins are cool too, although we are also trying to figure out the best way to use the tool. Ref https://github.com/babel/babel.github.io/pull/898#discussion_r76547376

And if you'd like to look over the blog post that would be 👍 too
